_studio/shared/umc/codec/mpeg2_dec/src/umc_mpeg2_bitstream.cpp

```diff
@@ -295,6 +296,15 @@ namespace UMC_MPEG2_DECODER
         if (CheckBSLeft())
             throw mpeg2_exception(UMC::UMC_ERR_INVALID_STREAM);
 
+        uint32_t width_in_MBs = (uint32_t)(mfx::align2_value<uint32_t>(slice.GetSeqHeader().horizontal_size_value, 16) / 16);
```

Comment:
I think explicit cast is not needed, just use unsigned literals:
```
uint32_t width_in_MBs = mfx::align2_value<uint32_t>(slice.GetSeqHeader().horizontal_size_value, 16) / 16u;
```

Reply:
Done